src/components/Card/Card.stories.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { Stories } from '@storybook/addon-docs';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import { Card, CardType } from './';
+import { IconName } from '../Icon';
+
+export default {
+    title: 'Card',
+    parameters: {
+        docs: {
+            page: (): JSX.Element => (
+                <main>
+                    <article>
+                        <section>
+                            <h1>Card</h1>
+                            <p>TBD</p>
+                        </section>
+                        <section>
+                            <Stories includePrimary title="" />
+                        </section>
+                    </article>
+                </main>
+            ),
+        },
+    },
+} as ComponentMeta<typeof Card>;
+
+const Base_Card_Story: ComponentStory<typeof Card> = (args) => (
```

Comment:
Add a custom `Card` story too where the content is defined via props - maybe use the alternate design as the sample?